libs/linkBankOperations.spec.js

```diff
@@ -121,12 +122,20 @@ describe('linker', () => {
     cozyClient.data.query.mockReturnValue(Promise.resolve(operations))
 
     const defaultOptions = {
-      minAmountDelta: 1, maxAmountDelta: 1,
-      pastWindow: 1, futureWindow: 1
+      minAmountDelta: 1,
+      maxAmountDelta: 1,
+      pastWindow: 1,
+      futureWindow: 1
+    }
+
+    function updateOperation (doctype, id, attributes) {
+      const operation = operations.find(operation => operation._id === id)
+      Object.assign(operation, attributes)
+      return Promise.resolve(operation)
```

Comment:
Since here we mutate objects that could be used for other tests, I would add a beforeEach to reset operations for every test.